Package.swift

```diff
@@ -63,6 +63,7 @@ let package = Package(
         .target(
             name: "SmithyRetries",
             dependencies: [
+                "SmithyRetriesAPI",
```

Comment:
This was an omission from the retries project that can lead to compile failures under certain conditions.